src/controllers/rating.controller.js

```diff
@@ -60,6 +61,41 @@ class RatingsController {
       return responseGenerator.sendError(res, 500, error.message);
     }
   }
+
+  /**
+   * Get all bookmarked articles
+   * @async
+   * @param {object} req - Request object
+   * @param {object} res - Response object
+   * @return {json} Returns containing all ratings of an article
+   * @static
+   */
+  static async getRatings(req, res) {
+    const { articleId } = req.params;
+
+    const averageRating = await BaseRepository.averageRating(
+      articleId,
+      db.Rating,
+      'articleId',
+      'ratings'
+    );
+    const [response] = averageRating;
+    if (!response) {
+      return responseGenerator.sendError(
+        res,
+        404,
+        'This article has not been rated yet'
+      );
+    }
+
+    const summary = {
+      articleId: response.articleId,
+      averageRating: Number(response.avgRating).toFixed(2),
+      totalNumberOfRatings: Number(response.totalCount).toFixed(2)
+    };
+
+    return responseGenerator.sendSuccess(res, 200, null, { ...summary });
```

Comment:
The signature of the response generator takes data as the third argument  - `sendSuccess(res, statusCode, data, message = '', metadata = '')`. the summary should be the third argument